merkle/objhasher/objhasher.go

```diff
@@ -16,42 +16,40 @@
 package objhasher
 
 import (
-	"crypto/sha256" // Use SHA256 to match ObjectHash.
+	"crypto"
 
 	"github.com/benlaurie/objecthash/go/objecthash"
+	"github.com/google/trillian/merkle"
+	"github.com/google/trillian/merkle/maphasher"
+	"github.com/google/trillian/merkle/rfc6962"
 )
 
-// ObjectHasher uses ObjectHash to compute leaf hashes.
-var ObjectHasher = &objhasher{}
+// ObjectMapHasher uses ObjectHash to compute leaf hashes in a sparse merkle tree.
+var ObjectMapHasher merkle.MapHasher = &objmaphasher{
+	// Use SHA256 to match ObjectHash.
+	MapHasher: maphasher.New(crypto.SHA256),
+}
+
+// ObjectLogHasher uses ObjectHash to compute leaf hashes in a dense merkle tree.
+var ObjectLogHasher merkle.LogHasher = &objloghasher{
+	// Use SHA256 to match ObjectHash.
+	LogHasher: rfc6962.New(crypto.SHA256),
```

Comment:
Can we avoid baking in the RFC 6962 hasher? And same for the map, where we are likely to have multiple ones.